internal/tiltfile/os/os.go

```diff
@@ -77,18 +85,70 @@ func osName() string {
 	return "posix"
 }
 
-func environ() (starlark.Value, error) {
-	env := os.Environ()
-	result := starlark.NewDict(len(env))
-	for _, e := range os.Environ() {
-		pair := strings.SplitN(e, "=", 2)
-		err := result.SetKey(starlark.String(pair[0]), starlark.String(pair[1]))
-		if err != nil {
-			return nil, err
-		}
+func getenv(t *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var varname starlark.Value
+	var defaultVal starlark.Value = starlark.None
+	err := starkit.UnpackArgs(t, fn.Name(), args, kwargs,
+		"varname", &varname,
+		"value?", &defaultVal,
```

Comment:
```suggestion
		"default?", &defaultVal,
```

Reply:
ahhhh...good catch! this doc
https://docs.python.org/2/library/os.html
says it's value but this doc says it's default
https://docs.python.org/3/library/os.html
and when i tried it out in a real python interpreter, they all only accepted 'default' :rolledeyes:

Reply:
Weird. Windows weird.